docs/source/developer_guide/dependencies.rst

```diff
@@ -74,6 +74,13 @@ To add an estimator with a soft dependency, ensure the following:
    utility should be used in ``__init__``. Both the warning and constructor call should use the ``package_import_alias`` argument for this.
 *  If the soft dependencies require specific python versions, the ``python_version``
    tag should also be populated, with a PEP 440 compliant version specification ``str`` such as ``"<3.10"`` or ``">3.6,~=3.8"``.
+*  Skip the corresponding doctests.  To do this add a ``# doctest: +SKIP`` to the end of each line in the doctest to skip.
+   If concerned about reducing testing coverage, can add the doctest back in the testing file. See arima as an example.
+*  Decorate all tests for the estimator with the soft dependency with a ``@pytest.mark.skipif(...)`` conditional on a check to ``_check_soft_dependencies``
```

Comment:
Maybe also more general, "all tests that import the soft dependency". Also mention that it should not be imported in the module, it should be imported inside the test.